<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLayoutSettings</class>
 <widget class="QDialog" name="DialogLayoutSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creation a layout</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../share/resources/icon.qrc">
    <normaloff>:/icon/64x64/icon64x64.png</normaloff>:/icon/64x64/icon64x64.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBoxPaperSize">
         <property name="title">
          <string>Paper size</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Templates:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxTemplates"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="2">
             <widget class="QComboBox" name="comboBoxPaperSizeUnit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>124</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelWidth">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelHeight">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxPaperHeight">
              <property name="minimumSize">
               <size>
                <width>94</width>
                <height>0</height>
               </size>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="maximum">
               <double>99999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxPaperWidth">
              <property name="minimumSize">
               <size>
                <width>94</width>
                <height>0</height>
               </size>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="maximum">
               <double>99999.990000000005239</double>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="labelSizeDescription">
              <property name="text">
               <string notr="true">TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QToolButton" name="toolButtonPortrate">
                <property name="text">
                 <string notr="true">...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../share/resources/icon.qrc">
                  <normaloff>:/icon/16x16/portrait.png</normaloff>:/icon/16x16/portrait.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButtonLandscape">
                <property name="text">
                 <string notr="true">...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../share/resources/icon.qrc">
                  <normaloff>:/icon/16x16/landscape.png</normaloff>:/icon/16x16/landscape.png</iconset>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <property name="autoRaise">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxRotate">
         <property name="title">
          <string>Rotate workpiece</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Rotate by</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxIncrease">
            <property name="currentIndex">
             <number>21</number>
            </property>
            <item>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">36</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">40</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">45</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">60</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">72</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">90</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">180</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>degree</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxCreationOptions">
       <property name="title">
        <string>Creation options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxLayoutWidth">
            <property name="minimumSize">
             <size>
              <width>94</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999.990000000005239</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelLayoutWidth">
            <property name="text">
             <string>Layout width:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelShiftLength">
            <property name="text">
             <string>Shift length:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxShift">
            <property name="minimumSize">
             <size>
              <width>94</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <double>99999.990000000005239</double>
            </property>
            <property name="value">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QComboBox" name="comboBoxLayoutUnit"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string notr="true">×2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBoxCase">
          <property name="title">
           <string>Principle of choosing the next workpiece</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="radioButtonThreeGroups">
               <property name="text">
                <string>Three groups: big, middle, small</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroupPrinciple</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonTwoGroups">
               <property name="text">
                <string>Two groups: big, small</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroupPrinciple</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonDescendingArea">
               <property name="text">
                <string>Descending area</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroupPrinciple</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../share/resources/icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogLayoutSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogLayoutSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupPrinciple"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
